### ISSUE TITLE
Coach can remove/delete a session with no attendance (and corresponding attendance records)

### ISSUE THREAD

**Issue by CritiqalPete** [AmericaSCORESBayArea/AmericaScores-CoachApp#211]
Use Case:
San Jose SD coaches are still only beginning to define their schedules already 5 weeks into program. They are capable of using the single session-add in the app.
Program leadership will want them or their coordinators to remove sessions that didnt happen to avoid false absences being counted.

UX:
1. Coach selects a session from the Sessions or Team Sessions view. 
2. A Delete Session action is revealed through a gesture  or UI element.
3. A message appears "Warning:Removing a session will delete any recoded attendance data. [Cancel][Continue]"
4. The coach confirms the day/date they are deleting and the view refreshes after a short animation and message:
"Session Removed"

*System Requirements*
Will need to research what we can delete. Session record only? dependent attendance records also?

**Comment by CritiqalPete** [comment]
@MarcHenriA can you confirm that Salesforce will allow a Session delete if the are some attendance records with TRUE.
IF yes, then we can check that the numbers are preserved in reports (which they should be) and then we can go ahead and allow a delete without cascading and with less of the complexity in this current design (maybe just a reminder that attendance values will be preserved)

**Comment by MarcHenriA** [comment]
I ran a test on workbench by creating a session for one of the test team season. I also added 3 attendance records with TRUE. Then I tried to delete the session and I observed that not only Salesforce allows a Session with attendance records TRUE to be deleted but also once the session is deleted the attendance is also deleted so the 3 attendance that I created for that session were also deleted.

To answer shortly: Yes I confirm
